types/react-native-privacy-snapshot/index.d.ts

```diff
@@ -0,0 +1,12 @@
+// Type definitions for react-native-privacy-snapshot 1.0
+// Project: https://github.com/kayla-tech/react-native-privacy-snapshot#readme
+// Definitions by: Matthew Bajorek <https://github.com/mattbajorek>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+interface PrivacySnapshot {
+    enabled: (isEnabled: boolean) => void;
```

Comment:
Rather than defining a `const` and this interface, you can just directly do `export function enabled(isEnabled: boolean): void`.

Reply:
Great suggestion! I have updated the code.